cai_causal_graph/time_series_causal_graph.py

```diff
@@ -441,7 +441,7 @@ def extend_graph(
             # Start from 1 as 0 is already defined.
             # We cannot start directly from maxlag as it may be possible that not all the nodes from 1 to -maxlag are
             # defined (as they were not needed in the minimal graph).
-            maxlag = minimal_graph.maxlag
+            maxlag = minimal_graph.max_backward_lag if minimal_graph.max_backward_lag is not None else None
```

Comment:
This can just be `maxlag = minimal_graph.max_backward_lag` as if None you set to None anyways.